web/src/main/java/se/inera/intyg/intygstjanst/web/service/impl/CertificateEventServiceImpl.java

```diff
@@ -69,6 +71,9 @@ private boolean created(String certificateId) {
     private boolean revoked(String certificateId) {
         final var response = getCertificateXmlService.get(certificateId);
         final var certificateXml = decodeXml(response.getXml());
+        if (response.getRecipient().getSent() != null) {
```

Comment:
Since Recipient can be null this will cause a nullpointer. Either just check for recipient or extend the null check

```java
response.getRecipient() != null && response.getRecipient().getSent() != null
```